source/client-side-encryption/corpus/corpus-encrypted.json

```diff
@@ -11,7 +11,7 @@
     "allowed": true,
     "value": {
       "$binary": {
-        "base64": "AgFkgAAAAAAAAAAAAAAAAAABM9KQ+Ph2lxyYUIV+Tu7gkrFndur280S5r43432bT+Q0WKKGSlu7s4anRPCvpwSyC7zP2VOD9ggBp29OvXsjm6A==",
```

Comment:
Random values all changed during regeneration. It seems silly to include them the expectation data when there's really nothing to compare them against, I could remove them in a follow up commit if it matters.